server/src/db/dal/init-db.ts

```diff
@@ -2,49 +2,49 @@ import { addCategory } from "./categories.dal";
 import { Category } from "../schemas/category.schema";
 
 const categories: Category[] = [{
```

Comment:
In order to help with the reuse for initializing all items (haven't fully checked it, I just wrote it here, you might find few minor mistakes):
Create an interface called `CategoryWithItems` extends `Category`, and it'll have the field `itemNames: string[]`.
Each category here will have its relevant items.

In `initCategoriesDb` do: 
```
await Promise.all(
        categories.map(async ({name, imageUrl, sentenceBeginning}) => await addCategory({name, imageUrl, sentenceBeginning}))
);
```

In `initItemsDb` do:
```
// categories defined in `init-db.ts`
const categoriesWithItemNames: CategoryWithItems[] = categories.map((categoryWithItems: CategoryWithItems) => ({
  ...categoryWithItems,
  id: (await CategoryModel.find({name: categoryWithItems.name}).exec())!.id
}));

await Promise.all(
  categoriesWithItemNames.map(async ({id, name, itemNames}) => {
    await initItemsForCategoryDb(itemNames, id);
    console.log(`Finished initializing all items in DB for ${name} category`);
}));
```

Then remove ALL unused code :)